tips/templates/tips/style.css

```diff
@@ -0,0 +1,648 @@
+    margin-bottom: 0px;
+}
+
+.comment-list .media .media-body ul li a {
+    color: #919ea8;
+    font-size: 18px;
+    font-weight: 500;
+}
+
+.comment-list .media .media-body ul li a:hover {
+    color: #36a0e7;
+}
+
+.comment-list .media .media-body ul li a.reply-btn {
+    color: #49545b;
+    text-decoration: underline;
+}
+
+.comment-list .media .media-body ul li a.reply-btn:hover {
+    color: #36a0e7;
+}
+
+.comment-list .media .media-body ul li {
+    font-size: 18px;
+    padding-right: 15px;
+    color: #919ea8;
+    font-weight: 500;
+}
+
+.comment-form {
+    float: left;
+    width: 100%;
+    padding: 30px 45px;
+}
+
+.comment-form h4 {
+    font-weight: 300;
+    font-size: 28px;
+    color: #3b4952;
+    margin-bottom: 40px;
+}
+
+.comment-form .form-control {
+    border-radius: 0px;
+    background: #f1f4f6;
+    border: none;
+    height: 50px;
+    color: #4a555c;
+    font-size: 16px;
+}
+
+.comment-form .form-control::-webkit-input-placeholder {
+    color: #4a555c;
+}
+
+.comment-form .form-control:-moz-placeholder {
+    color: #4a555c;
+}
+
+.comment-form .form-control::-moz-placeholder {
+    color: #4a555c;
+}
+
+.comment-form .form-control:-ms-input-placeholder {
+    color: #4a555c;
+}
+
+.comment-form textarea.form-control {
+    height: auto;
+    min-height: 200px;
+    resize: none;
+}
+
+.comment-form form {
+    margin-bottom: 40px;
+}
+
+.vt-post.post .author-info ul li {
+    padding-left: 15px;
+    padding-right: 15px;
+    float: none;
+}
+
+.vt-post.post .author-info {
+    border-radius: 0px 0px 0px 3px;
+    border-bottom: none;
+    border-right: 1px solid #ddd;
+    padding: 15px 12px 15px 12px;
+}
+
+.vt-post.post .post-img a img {
+    border-radius: 3px 0px 0px 0px;
+}
+
+.vt-post.post .caption {
+    padding: 25px 0px;
+}
+
+.post .caption h3 {
+    margin: 0px 0px 20px 0px;
+    color: #36a0e7;
+    font-weight: 300;
+    font-size: 34px;
+    line-height: 42px;
+}
+.md-heading {
+    font-size: 24px !important;
+    line-height: 36px !important;
+    margin-bottom: 15px !important;
+}
+
+.pagination>.active>a,
+.pagination>.active>a:focus,
+.pagination>.active>a:hover,
+.pagination>.active>span,
+.pagination>.active>span:focus,
+.pagination>.active>span:hover {
+    background-color: #3b4952;
+    border-color: #3b4952;
+}
+
+.pagination>li>a,
+.pagination>li>span {
+    color: #2c3840;
+    margin: 0px 5px;
+    border-radius: 3px;
+    -webkit-box-shadow: 0px 1px 3px 0px rgba(44, 56, 64, 0.2);
+    -moz-box-shadow: 0px 1px 3px 0px rgba(44, 56, 64, 0.2);
+    box-shadow: 0px 1px 3px 0px rgba(44, 56, 64, 0.2);
+    border: none;
+    font-size: 16px;
+}
+
+.pagination>li>a:focus,
+.pagination>li>a:hover,
+.pagination>li>span:focus,
+.pagination>li>span:hover {
+    background-color: #e74c3c;
+    border-color: #e74c3c;
+    color: #fff;
+}
+
+.pagination-wrap {
+    width: 100%;
+    float: left;
+    margin-bottom: 35px;
+}
+
+.pagination {
+    margin: 0px;
+}
```

Comment:
It looks like a very long CSS that we don't really use all of it, can you remove all the unnecessary code
Also empty line at the end

Reply:
A lot of it is  maybe necessary for adjuasnments I want to do later